src/components/victory-chart/victory-chart.jsx

```diff
@@ -207,14 +246,60 @@ export default class VictoryChart extends React.Component {
     return {axisComponents, categories, domain, flipped, scale, stringMap};
   }
 
-  // the old ones were bad
   getNewChildren(props, childComponents, baseStyle) {
+    const nodesWillExit = this.state && this.state.nodesWillExit;
+    const nodesWillEnter = this.state && this.state.nodesWillEnter;
+    const nodesShouldEnter = this.state && this.state.nodesShouldEnter;
+    const childrenTransitions = this.state && this.state.childrenTransitions;
+
     const calculatedProps = this.getCalculatedProps(props, childComponents);
     return childComponents.map((child, index) => {
+      let animate = child.props.animate || props.animate;
+      let exitingNodes = null;
+      let enteringNodes = null;
+
+      if (animate && nodesWillExit) {
+        exitingNodes = childrenTransitions[index] && childrenTransitions[index].exiting;
+        animate = exitingNodes ?
+          assign({}, animate || {}, {
```

Comment:
And to clarify, do you envision something other than fade in/out duration?  That may be possible, but I'm struggling to find an elegant way of exposing that to the user.